src/middlewares/authMiddleware.js

```
@@ -0,0 +1,25 @@
+const jwt = require('jsonwebtoken');
+const JWT_SECRET = process.env.JWT_SECRET;
```

You read JWT_SECRET directly from env but do not handle the case where it is undefined. If JWT_SECRET is missing jwt.verify will throw unclear errors. Validate and fail fast (log or throw) during app startup, or at least return a 500 here with a clear message. This relates to security and reliability: handlers relying on an undefined secret are fragile. Consider `if (!JWT_SECRET) { console.error('Missing JWT_SECRET'); /* respond or throw */ }`.